src/js/components/Button/Button.js

```diff
@@ -92,10 +92,12 @@ const getIconColor = (paths = [], theme, colorProp, kind) => {
   return result[1] || undefined;
 };
 
+const tipString = tip => {
```

Comment:
Aren't you missing a `}`?

Also, I think we can remove the function and just make it a constant variable.

Reply:
ahh, that's what I get for approving GitHub suggestions directly. You're right, now that the assignment is a ternary, it's short enough to inline in the default assignment. This change has been made.